.ci/scripts/install-tools.bat

```diff
@@ -0,0 +1,10 @@
+set GOPATH=%WORKSPACE%
+set MAGEFILE_CACHE=%WORKSPACE%\.magefile
+set PATH=%WORKSPACE%\bin;C:\ProgramData\chocolatey\bin;%PATH%
+
+choco install curl -y --no-progress --skipdownloadcache
```

Comment:
Is it possible to use PowerShell scripting? Then we can use the native `Invoke-WebRequest` cmdlet to download. Maybe a future optimization.

Reply:
the problem with PowerShell it is that to make something simple it is really verbose, also you have to manage to control the errors in the script execution, and finally, the error messages are most of the time cryptic and difficult to know what the hell happened. I will avoid it at least until we have something working

Reply:
```suggestion
where /q curl
IF ERRORLEVEL 1 (
 choco install curl -y --no-progress --skipdownloadcache
)
```